Weather App/view/ForecastWeatherTableViewCell.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ForecastWeatherTableViewCell.swift
+//  Weather App
+//
+//  Created by Shikhar Sharma on 17/03/22.
+//
+
+import UIKit
+
+public class ForecastWeatherTableViewCell: UITableViewCell {
+    
+    @IBOutlet public weak var DayLabel: UILabel!
```

Comment:
Are you not following camelCase for property names?

Reply:
forgot to follow in some outlets and action,
fixed